kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/services/ProductOrderRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.good.patterns.orders.ordering.services;
+
+import com.kodilla.good.patterns.orders.ordering.PlacedProductOrder;
+
+public class ProductOrderRepository implements OrderRepository {
```

Comment:
I have problem with naming. Interface OrderRepository allows us to place orders of products (boolean create(PlacedProductOrder placedOrder))
So in my opinion this class can say something about implementation. Let assume that in this case we store all information in memory (instead of using database), so I propose to call it InMemoryOrderRepository